src/main/java/com/groupbyinc/quickstart/controller/NavigationController.java

```diff
@@ -87,10 +87,14 @@
     UrlBeautifier.createUrlBeautifier("default");
     defaultUrlBeautifier = UrlBeautifier.getUrlBeautifiers().get("default");
     defaultUrlBeautifier.addRefinementMapping('s', "size");
+    defaultUrlBeautifier.addRefinementMapping('b', "brand");
+    defaultUrlBeautifier.addRefinementMapping('c', "categories.1");
+    defaultUrlBeautifier.addRefinementMapping('t', "categories.2");
+    defaultUrlBeautifier.addRefinementMapping('g', "categories.3");
     defaultUrlBeautifier.setSearchMapping('q');
     defaultUrlBeautifier.setAppend("/index.html");
-    defaultUrlBeautifier.addReplacementRule('/', ' ');
-    defaultUrlBeautifier.addReplacementRule('\\', ' ');
+    defaultUrlBeautifier.addReplacementRule('/', '\u002F');
+    defaultUrlBeautifier.addReplacementRule('\\', '\u005C\u005C');
```

Comment:
Is Unicode what we want here? How does this relate to URL encoding in general? I may lack the req'd familiarity with URL encoding to be the judge